eng/code-quality-reports/src/main/resources/checkstyle/checkstyle-suppressions.xml

```diff
@@ -610,4 +610,19 @@
   <!-- Suppress checks in azure-core-experimental -->
   <suppress checks="com.azure.tools.checkstyle.checks.ExternalDependencyExposedCheck"
             files="com.azure.core.experimental.jsonpatch.(JsonPatchDocumentSerializer|JsonPatchOperationSerializer).java"/>
+
+
+  <!-- Suppress checks all azure-spring-cloud MissingJavadocMethod and MissingJavadocType -->
+  <suppress checks="MissingJavadocMethod" files=".*[/\\]com[/\\]azure[/\\]spring[/\\].*"/>
+  <suppress checks="MissingJavadocType" files=".*[/\\]com[/\\]azure[/\\]spring[/\\].*"/>
+  <suppress checks="JavadocPackage" files=".*com[/\\]azure[/\\]spring[/\\].*\.java"/>
+
+  <!-- Suppress checks all azure-spring-cloud -->
+  <suppress checks="com.azure.tools.checkstyle.checks.(GoodLoggingCheck|ThrowFromClientLoggerCheck)" files=".*[/\\]com[/\\]azure[/\\]spring[/\\].*"></suppress>
+  <suppress checks="com\.azure\.tools\.checkstyle\.checks\.(JavadocThrowsChecks|ThrowFromClientLoggerCheck|ExternalDependencyExposedCheck|EnforceFinalFieldsCheck)"
+            files=".*[/\\]com[/\\]azure[/\\]spring[/\\].*"/>
+  <suppress checks="com.azure.tools.checkstyle.checks.BlacklistedWordsCheck"
```

Comment:
What's the word that's blacklisted here.

Reply:
URL